app/javascript/packs/application.js

```diff
@@ -30,38 +30,28 @@ import ClassicEditor from '@ckeditor/ckeditor5-build-classic'
 // Below import resolves to '/app/javascript/components/HomePageContent.js'
 import HomePageContent from 'components/HomePageContent'
 import Covid19PageContent from 'components/covid19/Covid19PageContent'
-import { logPageView, logClick } from 'lib/metrics-api'
+import { logPageView, logClick, log } from 'lib/metrics-api'
 import createTracesAndLayout from 'lib/kernel-functions'
 
 document.addEventListener('DOMContentLoaded', () => {
-  if (document.getElementById('home-page-content')) {
-    // Logs only page views for faceted search UI
-    //
-    // If/when Mixpanel is extended beyond home page, remove study name from
-    // appPath in metrics-api.js at least for non-public studies to align with
-    // Terra on identifiable data that we want to omit from this logging.
-    logPageView()
+  // Logs only page views for faceted search UI
+  logPageView()
+
+  $(document).on('click', 'body', event => {
```

Comment:
I'm sure the answer is "yes", but this doesn't interfere with any other click handlers?  We have a lot...

Reply:
Yup, I just re-verified several click handlers, including those added via `$(document).on('click'...`.